Feat: Create About.jsx

HeroicHitesh opened the pull request:
### Description

- Create `About.jsx` for About section in `home` folder
- Add  `/about` route in `Routes.js`
- Move `Home.jsx` to `home` folder
- update `react-scripts` dependency from version `3.4.1` to `^3.4.3` in `package.json`

Fixes #96 

### Type of Change:

- Code
- User Interface

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

### Screenshot of change:

<img width="1440" alt="Screenshot 2020-09-17 at 12 50 38 PM" src="https://user-images.githubusercontent.com/37622734/93433495-7bb65400-f8e4-11ea-9eb6-47e1412a3ba0.png">

Commit:
Feat: Create About.jsx

Create About.jsx for About section
Add  route in Routes.js
Move Home.jsx to home folder

Fixes #96

mtreacy002 reviewed (changes requested):
@HeroicHitesh , I can't see the image being displayed when I run the app. It looks like you need to add the image file locally inside the `home` folder and refer to it as the `src` address.

Actually..... it's now visible (after waiting for a while). I wonder if it's better to add image to folder or use online refence 🤔

Perhaps we can keep it as it is for now, and only change it later when it becomes an issue.

Other than that, this PR looks good 👍  and I noticed that you've made the navbar looks `prettier`, which is awesome ✨✨. 
But I'd like to ask you to change one thing. It's not a fault from your side, I just have a change of mind with the url link to `Read more about the projeect here...`. Initially I put the BridgeInTech repo as link since I'm thinking as a developer. But our user might not be interested on contributing to BridgeInTech project as developer, instead they want to use BridgeInTech as a user (organization/mentor/mentee). In this situation, an intro to what the application is about and benefit seems to be more useful for our user. 

For that purpose, can you please change the link to `https://medium.com/anitab-org-open-source/bridgeintech-by-anitab-org-85745853865d?source=friends_link&sk=fcb616ef0ac7e1e85ac6b58b513a56ee` instead? This is my medium blog post for [Introduction to BridgeInTech](https://medium.com/anitab-org-open-source/bridgeintech-by-anitab-org-85745853865d?source=friends_link&sk=fcb616ef0ac7e1e85ac6b58b513a56ee) from an application overview.

I really appreciate your help. 👍

HeroicHitesh committed:
Add medium article link to Read more...

Add [Medium article link](https://medium.com/anitab-org-open-source/bridgeintech-by-anitab-org-85745853865d?source=friends_link&sk=fcb616ef0ac7e1e85ac6b58b513a56ee) to `a` tag of `Read more about the project here...`

HeroicHitesh commented:
> @HeroicHitesh , I can't see the image being displayed when I run the app. It looks like you need to add the image file locally inside the home folder and refer to it as the src address.
>
> Actually..... it's now visible (after waiting for a while). I wonder if it's better to add image to folder or use online refence 🤔
>
> Perhaps we can keep it as it is for now, and only change it later when it becomes an issue.

@mtreacy002 I read this in one of the GitHub repo's PR reviews that it's better to use online reference instead of statically adding content, to reduce the total repository size. 
Also, I have one more suggestion if we are using AWS we can store static content in S3, which will not increase repository size and I hope there will be no display delay as it Amazon😅

> For that purpose, can you please change the link to `https://medium.com/anitab-org-open-source/bridgeintech-by-anitab-org-85745853865d?source=friends_link&sk=fcb616ef0ac7e1e85ac6b58b513a56ee` instead? This is my medium blog post for Introduction to BridgeInTech from an application overview.

Done👍

mtreacy002 commented on package.json:
```diff
@@ -12,7 +12,7 @@
     "react-dom": "^16.13.1",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "3.4.1"
+    "react-scripts": "^3.4.3"
```
any reason why we need to update the version here? just curious.

HeroicHitesh replied:
I ran `npm install` and it showed some warnings so I ran `npm audit fix` and that updated this version which I guess will reduce any vulnerabilities we may find in the future. Also, adding `^` helps use any version greater than or equal to the mentioned one

mtreacy002 replied:
I see. In that case, can you please add to the PR description under `Type of Change` that extra info: "update react-scripts dependency from version 3.4.1 to 3.4.3" follows by instruction for other contributors to update this dependency locally

HeroicHitesh replied:
Done

mtreacy002 commented on src/home/About.jsx:
```diff
@@ -0,0 +1,52 @@
+          allies.
+        </p>
+        <a
+          href="https://medium.com/anitab-org-open-source/bridgeintech-by-anitab-org-85745853865d?source=friends_link&sk=fcb616ef0ac7e1e85ac6b58b513a56ee"
```
Thank you for accommodating with the request, @HeroicHitesh 👍

mtreacy002 reviewed (commented):
Hi @HeroicHitesh, just need one more confirmation from you.

mtreacy002 commented:
> @mtreacy002 I read this in one of the GitHub repo's PR reviews that it's better to use online reference instead of statically adding content, to reduce the total repository size.

Makes sense...

> Also, I have one more suggestion if we are using AWS we can store static content in S3, which will not increase repository size and I hope there will be no display delay as it Amazon😅

That's also my wish, @HeroicHitesh, for this purpose and also for the purpose of having reliable remote servers .... cc @anitab-org/bridgeintech-maintainers and @anitab-org/bridgeintech-managers 🙏

mtreacy002 reviewed (approved):
You did a great job, @HeroicHitesh on your First-Timers-Only issue. I'm really happy that you''re contributing to the project 🎉. Look forward to your next contribution 😉

HeroicHitesh commented:
> You did a great job, @HeroicHitesh on your First-Timers-Only issue. I'm really happy that you''re contributing to the project 🎉. Look forward to your next contribution 😉

Thank you for the kind words, looking forward to contributing to more :electron: `React`ive issues to improve the platform

mtreacy002 commented:
> > You did a great job, @HeroicHitesh on your First-Timers-Only issue. I'm really happy that you''re contributing to the project 🎉. Look forward to your next contribution 😉
> 
> Thank you for the kind words, looking forward to contributing to more :electron: `React`ive issues to improve the platform

Don't forget there're many ways you could contribute to the project, as mentioned on [zulip post here](https://anitab-org.zulipchat.com/#narrow/stream/223071-newcomers/topic/Want.20to.20contribute.3F/near/210336297). Perhaps start a discussion about the design on the #bridge-in-tech stream on the Navbar improvement you mentioned before? 😁

HeroicHitesh commented:
> Don't forget there're many ways you could contribute to the project, as mentioned on zulip post here. 

True. Actually I am more inclined towards resolving all the React issues, so I mentioned that apart from that I'll surely contribute wherever I can.

> Perhaps start a discussion about the design on the #bridge-in-tech stream on the Navbar improvement you mentioned before? 😁

I guess you confused me with `Prathmesh Khandelwal` he started Navbar discussion [here](https://anitab-org.zulipchat.com/#narrow/stream/237630-bridge-in-tech/topic/New.20Issues/near/210322643)
Or are you talking about restructuring the project components which I mentioned [here](https://github.com/anitab-org/bridge-in-tech-web/issues/96#issuecomment-693834761)

mtreacy002 commented:
> > Don't forget there're many ways you could contribute to the project, as mentioned on zulip post here.
> 
> True. Actually I am more inclined towards resolving all the React issues, so I mentioned that apart from that I'll surely contribute wherever I can.
> 
> > Perhaps start a discussion about the design on the #bridge-in-tech stream on the Navbar improvement you mentioned before? 😁
> 
> I guess you confused me with `Prathmesh Khandelwal` he started Navbar discussion [here](https://anitab-org.zulipchat.com/#narrow/stream/237630-bridge-in-tech/topic/New.20Issues/near/210322643)
> Or are you talking about restructuring the project components which I mentioned [here](https://github.com/anitab-org/bridge-in-tech-web/issues/96#issuecomment-693834761)

aaa... yes, sorry, got mixed up 😊.

Anyway, any kind of participation is welcomed 😉. There are few other available issues [here](https://github.com/anitab-org/bridge-in-tech-web/issues) if you're interested. Just ping me if you find one so I can assign you to it 😁

HeroicHitesh commented:
> There are few other available issues here if you're interested. 

I have no idea about testing in React but would love to learn something about it, so can you redirect me to some useful resources for me to get started with contributing to #33 

> Just ping me if you find one so I can assign you to it 😁

😅Sorry for messaging here, in Zulip PMs are disabled, so I wasn't able to ping you there.

mtreacy002 commented:
@HeroicHitesh , on our AnitaB.Org Open Source zulip, the PM is disabled. However, you don't need to use PM at all to ping me. As part of Open Source value, we encourage open communication. So you can ping me (or anyone you want to reach out) on the channel/topic you want to have discussion on. 

The benefit of pinging using zulip is it's much faster to reach out to member you ping instead of using github. Because zulip app also available on mobile phone and its notification works just like whatsapp or other Mobile Messaging application. 
Posting a comment on github normally will get noticed only when the person check their email, providing you use @<github username> (like @mtreacy002) to reach them on the post. If you don't put this tag+username, your post might get unnoticed until that person check the PR. 
You are lucky that I have the email notification to this repo is set as `on` so I get emailed on your message [above](https://github.com/anitab-org/bridge-in-tech-web/pull/100#issuecomment-694859990) eventhough you didn't put `@mtreacy002` on the post 😁. But I normally check my zulip first before my email.
